src/main/java/no/digipost/api/client/representations/PrintDetails.java

```diff
@@ -64,6 +69,12 @@ public PrintDetails(final PrintRecipient recipient, final PrintRecipient returnA
         this.nondeliverableHandling = nondeliverableHandling;
     }
 
+    public void setInstruction(List<PrintInstruction> instructions) {
+        if (!instructions.isEmpty()) {
```

Comment:
Kanskje heller legge denne til all-args-konstruktøren?

Nå blir det en konstruktør for påkrevde felter, en for ekstra felter og så en setter for det feltet som er ekstra-ekstra 😄

Reply:
Da blir det breaking change og ny major. Kan godt gjøre det altså, hvis du vil

Reply:
Beholder det som det er etter diskusjon 👍